src/server/common/model/answer/full-name/full-name.js

```diff
@@ -0,0 +1,160 @@
+import Joi from 'joi'
+import { AnswerModel } from '../answer-model.js'
+import { validateAnswerAgainstSchema } from '../validation.js'
+
+/** @import { AnswerViewModelOptions } from '../answer-model.js' */
+
+/**
+ * export @typedef {{
+ *  validation: {
+ *    firstName: {
+ *      maxLength?: { value: number, message: string },
```

Comment:
Make mandatory?

Reply:
Kept maxlength checks hardcoded in the fullname answer model so I'm removing it from the config to pass in